homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,79 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      This integration does not subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
```

Comment:
where is that done? `await solar_net.init_devices()` does not throw any `ConfigEntryNotReady` or similar exceptions and your not calling `coordinator.async_config_entry_first_refresh()`

Reply:
is there a reason though why you aren't doing a `coordinator.async_config_entry_first_refresh()` before continuing the setup though?

Reply:
It does that too in https://github.com/home-assistant/core/blob/954ac0d288d8768836803508a259060c4e22c2d3/homeassistant/components/fronius/__init__.py#L102 and https://github.com/home-assistant/core/blob/954ac0d288d8768836803508a259060c4e22c2d3/homeassistant/components/fronius/__init__.py#L205 and https://github.com/home-assistant/core/blob/954ac0d288d8768836803508a259060c4e22c2d3/homeassistant/components/fronius/__init__.py#L264